rust/src/transformations/polars/mod.rs

```diff
@@ -26,4 +26,40 @@ pub use scan_parquet::*;
 #[cfg(feature = "contrib")]
 mod sink_parquet;
 #[cfg(feature = "contrib")]
-pub use sink_parquet::*;
\ No newline at end of file
+pub use sink_parquet::*;
+#[cfg(feature = "contrib")]
+mod make_col;
+#[cfg(feature = "contrib")]
+pub use make_col::*;
+
+#[cfg(test)]
+mod test {
+    use crate::domains::{AtomDomain, ExprDomain, LazyFrameDomain, SeriesDomain, LazyFrameContext};
+    use crate::error::*;
+    use polars::prelude::*;
+
+    pub fn get_test_data() -> Fallible<(ExprDomain<LazyFrameContext>, LazyFrame, LazyFrameDomain)> {
```

Comment:
How do you feel about dropping `LazyFrameDomain` from the return? If you ever need it, can do `expr_domain.lazy_frame_domain`.